subsys/tfm/Kconfig

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+config PM_SINGLE_IMAGE
```

Comment:
Could this be an imply in the original definition of PM_SINGLE_IMAGE?


```
config PM_SINGLE_IMAGE
    imply if BUILD_WITH_TFM
```

Reply:
Not sure if `imply if` is a thing, but I can add a `default y if BUILD_WITH_TFM`.

Reply:
I think I'd actually prefer that it stays here, so the TFM-related stuff is mostly in the same place.

Reply:
Wait, the partition sizes are already in there. Sure, why not.

Reply:
I even moved the CMakeLists stuff into the partition_manager subsys, see if it's ok.